### DISCUSSION TITLE
light version

### DISCUSSION THREAD

**Discussion by FreeVB** [Live-Charts/LiveCharts2#610] [Ideas]
Have you considered releasing a light version? It is not cross-platform and does not use third-party controls.

**Comment by beto-rodriguez** [comment]
In theory we can consume the `LiveChartsCore` assembly and draw the chart from another graphics engine (currently the library only uses SkiaSharp).

What is the platform you are facing issues with?

**Comment by ajayre** [comment]
I took the codebase and removed all assemblies except for the core, skiasharp and winforms. Works well.

**Comment by FreeVB** [comment]
> 

I only use it on Windows platform. Thank you for your reply.

**Comment by FreeVB** [comment]
It's better to be like 1.0: only one DLL library is the most convenient.
